dataverse-webapp/src/main/java/edu/harvard/iq/dataverse/bannersandmessages/banners/dto/BannerMapper.java

```diff
@@ -123,7 +123,7 @@ private DataverseLocalizedBanner mapToLocalizedBanner(DataverseBanner banner, Da
         } else {
 
             try {
-                dataverseLocalizedBanner.setImage(StreamUtils.copyToByteArray(fuDto.getDisplayedImage().getStream()));
+                dataverseLocalizedBanner.setImage(StreamUtils.copyToByteArray(new ByteArrayInputStream(fuDto.getFile().getContent())));
```

Comment:
You can simply do dataverseLocalizedBanner.setImage(fuDto.getFile().getContent())

Reply:
Using `StreamUtils.copyToByteArray` throws IOException when something goes wrong, while simply `getContent()` does not. Wouldn't it be better to use `StreamUtils` afterall?

Reply:
Why? If something goes wrong later, some runtime exception will be thrown anyway.

Reply:
Which will make finding the root of the problem harder to find. Shouldn't you throw exceptions as soon as possible?

Reply:
And this code also doesn't make much sense since you already have the content so byte[] -> Stream -> byte[] so it's like turning 3 time right instead of taking 1 turn left :P.

Reply:
Yeah but what kind of exception would you throw now? You already have the content of the file.